qiita_ware/ebi.py

```diff
@@ -328,6 +316,7 @@ def generate_sample_xml(self):
                                              "/sra_1_3/SRA.sample.xsd"})
 
         for sample_name, sample_info in sorted(viewitems(self.samples)):
+            sample_info = dict(sample_info)
```

Comment:
object => it's simply so I can get all the elements and their values. Additionally this is helpful as I can remove the ones that are "required" and know which ones are left so they can be added as sample_attributes.